src/highlevel/forward_mode.jl

```diff
@@ -6,15 +6,20 @@ struct ForwardDiffJacobianCache{CO, CA, J, FX, X} <: AbstractMaybeSparseJacobian
     x::X
 end
 
+__getfield(c::ForwardDiffJacobianCache, ::Val{:jac_prototype}) = c.jac_prototype
+
 struct SparseDiffToolsTag end
 
+__standard_tag(::Nothing, x) = ForwardDiff.Tag(SparseDiffToolsTag(), eltype(x))
+__standard_tag(tag, _) = tag
```

Comment:
How is that supposed to work? Do you wrap this in a ForwardDiff.Tag somewhere else? Seems like it's missing the wrap

Reply:
I was assuming we wrap it while construction but it makes sense to do it here